packages.json

```diff
@@ -5264,5 +5264,22 @@
     "description": "ISAAC PRNG implementation on Nim",
     "license": "MIT",
     "web": "https://github.com/pragmagic/isaac/"
+  },
+  {
+    "name": "SDF.nim",
```

Comment:
I think it does. You would then need to rename your module to ``SDF.nim.nim`` and import it via ``import SDF.nim``, I dunno, maybe that works. Best way to check is to test it.

I'll merge this for now.